### PR TITLE
Moved RTIR to package structure

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
Ready

## Description
Moved RTIR into package structure.

### CODE REVIEW

**Comment by anara123** [comment]
@DeanArbel RTIR tests are failing

**Comment by DeanArbel** [comment]
@anara123 I noticed, I'm working on fixing it

File: Integrations/RTIR/RTIR.yml [hunk 1]
```diff
@@ -0,0 +1,575 @@
...
+- display: Server URL (e.g. http://192.168.0.1)
+  name: server
+  required: true
+  type: 0
+- display: Credentials
+  name: credentials
+  required: true
+  type: 9
+- defaultvalue: 'true'
+  display: Trust any certificate (unsecure)
```
Comment (Itay4) [inline Integrations/RTIR/RTIR.yml]:
```suggestion
  display: Trust any certificate (not secure)
```

**Comment by Itay4** [comment]
what is the referer for ?

**Comment by DeanArbel** [comment]
> what is the referer for ?

https://github.com/demisto/etc/issues/17615

**Comment by Itay4** [comment]
@DeanArbel its hard to see the changes done and review
lets split this to two PRs, one for the packagify and one for the improvement

**Comment by DeanArbel** [comment]
@Itay4 that was the plan, but I thought since the change was very minor (adding raw_response to one command and to add the referer header) we can do it one PR...

If you prefer I can remove those 2 changes...

File: Integrations/RTIR/RTIR_description.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+ Additional Parameters:
```
Comment (Itay4) [inline Integrations/RTIR/RTIR_description.md]: lets remove this as well, so it will be reviewed in the next pr

Comment (DeanArbel) [reply Integrations/RTIR/RTIR_description.md]: Right, missed that...

**Pull request merged**